src/framework/CODebugController.h

```diff
@@ -0,0 +1,22 @@
+//  Created by Mathieu Dutour on 01/05/2019.
+//  
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol CODebugControllerDelegate
+- (void)output:(NSString*)format args:(va_list)args;
+@end
+
+@interface CODebugController : NSObject
```

Comment:
Any overview docs?

Reply:
@mathieudutour Could you take a look at this?